src/app/faqs/faq-list/faq-list.component.html

```diff
@@ -0,0 +1,154 @@
+<div class="d-flex row-hl">
+  <!-- Category Selector -->
+  <div class="p-2 w-20">
+    <div *ngFor="let faqCategory of faqCategories">
+      <button
+        mat-raised-button
+        [color]="filterFaqCategory?.id == faqCategory.id ? 'accent' : ''"
+        class="card mb-3 w-100"
+        (click)="onClickCategory(faqCategory)"
+      >
+        <div
+          class="d-flex align-items-center justify-content-between flex-row p-3"
+        >
+          <i class="fa mr-4" [ngClass]="faqCategory.icon"></i>
+          <div class="d-flex flex-column align-items-center w-100">
+            <mat-card-title>
+              {{ faqCategory.title }}
+            </mat-card-title>
+            <span class="font-weight-normal line-height-normal text-wrap">
+              {{ faqCategory.description }}
+            </span>
+          </div>
+        </div>
+      </button>
+    </div>
+    <button
+      *ngIf="userIsAdmin"
+      mat-raised-button
+      color="primary"
+      class="w-100"
+      (click)="onClickAddCategory()"
+    >
+      Add Category
+    </button>
+  </div>
+
+  <!-- Categories -->
+  <div class="p-2 w-60 d-flex flex-column align-items-center">
+    <!-- Title -->
+    <h2 class="text-center">
+      Frequently Asked Questions
+    </h2>
+
+    <!-- Clear Filter Button -->
+    <button
+      class="mb-2"
+      mat-button
+      color="warn"
+      (click)="onClickCategory(null)"
+      *ngIf="faqCategories.length - getFilteredCategories().length"
+    >
+      {{ faqCategories.length - getFilteredCategories().length }} Category
+      hidden
+    </button>
+
+    <!-- Categories Accordian -->
```

Comment:
default to first category (highlight that category as active on left)  and have expansion panels for each faq instead of nested expansions